tables/mdm/mdm.go

```diff
@@ -43,9 +46,18 @@ type profileStatus struct {
 }
 
 type depStatus struct {
-	DEPCapable bool
+	DEPCapable  bool `json:"dep_capable"`
+	RateLimited bool `json:"rate_limited"`
```

Comment:
field names should technically not be exported/capitalized unless they are meant to be exported but the struct itself is not exported. Is this for unmarshaling?